libraries/AP_HAL_PX4/Util.cpp

```diff
@@ -183,6 +183,9 @@ void PX4Util::perf_count(perf_counter_t h)
 void PX4Util::set_imu_temp(float current)
 {
     if (!_heater.target || *_heater.target == -1) {
+#if defined(CONFIG_ARCH_BOARD_PX4FMU_V3)
```

Comment:
Why v3 only? All GCS install v2 in PH2.1... Actually I think this is the generic correct approach, if there is no target temperature or it is disabled then we should turn off the heater - it is very possible someone turns off the temperature target (by setting it to -1) before reaching it and expects it would disable the heater (and with current code it doesn't).